news/2017-12-03-osu-idol-2017-finals-community-voting-results.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "osu!idol 2017 Finals - Community Voting Results!"
+date: 2017-12-03 12:00:00 +0000
+---
+
+We have reached another year of osu!idol to an end. With a spectacular 12281 votes in total, this beats last year's voting by a lightyear! Thanks everyone for taking part in the [community voting](https://osu.ppy.sh/community/contests/57)! Now let's get to what we're all here for - results!
+
+![](/wiki/shared/news/2017-12-03-osu-idol-2017-finals-community-voting-results/banner.jpg)
+
+### Results
+
+**Congratulations to this year's osu!idol - [Renril](https://osu.ppy.sh/users/4955122)!** Their performance of **Mafumafu - Rinne Tensei** has gathered a total of 7207 votes which places them in first place! **In second place we have [Will Stetson](https://osu.ppy.sh/users/4909088)**'s performance of **Kaai Yuki - Ikanaide** accumulating 3344 votes! **Third place goes to [Thievley](https://osu.ppy.sh/users/4717672)** with their performance of **Flyleaf - All around me** netting 1730 votes! Applause for the outstanding talent from our winners.
```

Comment:
Probably add "and competitors" so that non-winners don't feel left out ?

Reply:
That would sound incredibly out of place. We're talking about finalists here.